CSS_files/components/grids/grids.css

```diff
@@ -0,0 +1,26 @@
+.grid-2{
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 1rem;
```

Comment:
You can use global variables for properties like margin, gap, and padding, like
--space-xsmall:0.5rem

Reply:
Ohh yes, I've created utility classes for margin and padding. As I was not using the gap property much, I didn't create a utility class for it. 
Thanks for the feedback, I will use global variables for such cases from now on.

Reply:
> You can use global variables for properties like margin, gap, and padding, like --space-xsmall:0.5rem

I have made the necessary changes, Thank you.